src/Cli/dotnet/NugetPackageDownloader/PackageSourceLocation.cs

```diff
@@ -13,18 +12,21 @@ internal class PackageSourceLocation
         public PackageSourceLocation(
             FilePath? nugetConfig = null,
             DirectoryPath? rootConfigDirectory = null,
-            string[] sourceFeedOverrides = null)
+            string[] sourceFeedOverrides = null,
+            string[] additionalSourceFeeds = null)
         {
             NugetConfig = nugetConfig;
             RootConfigDirectory = rootConfigDirectory;
-            ExpandLocalFeedAndAssign(sourceFeedOverrides);
+            SourceFeedOverrides = ExpandLocalFeed(sourceFeedOverrides);
```

Comment:
need some comment on what if both RootConfigDirectory and SourceFeedOverrides are used